### ISSUE TITLE
ko install failed by Go 1.18

### ISSUE THREAD

**Issue by sonatard** [google/ko#650]
```
$ go version
go version go1.18 darwin/arm64

$ go install github.com/google/ko@latest
...
...
# github.com/google/ko/internal/sbom
go/pkg/mod/github.com/google/ko@v0.10.0/internal/sbom/mod_1.18.go:28:16: dbi.UnmarshalText undefined (type *debug.BuildInfo has no field or method UnmarshalText)
```

**Comment by sonatard** [comment]
I found fix this issue
https://github.com/google/ko/pull/631

**Comment by imjasonh** [comment]
I'll probably cut a v0.11 some time tomorrow to pick up the fix, which is in https://github.com/google/ko/issues/625

In the meantime, you should be able to get around this with `go install github.com/google/ko@main` (note: not `@latest`), which will include the fix.

**Comment by mchmarny** [comment]
Thanks for pushing the new version @imjasonh. Seems to fixed the issues with install under go 1.18.

**Comment by mchmarny** [comment]
I spoke too soon. I am getting this now using `build` under 1.18

```shell
Error: failed to publish images: error building "ko://github.com/mchmarny/<repo>/cmd/app": 
could not parse Go build info: line 43: expected 2 columns for build setting; got 1
```
Here is the CI command I'm using

```shell
KO_DOCKER_REPO=gcr.io/<project>/<name> \
  GOFLAGS="-ldflags=-X=main.version=v0.7.8" \
  ko build ./cmd/app/ --image-refs ./image-digest --bare --tags v0.7.8,latest
```

**Comment by sonatard** [comment]
@imjasonh Thanks for releasing v0.11. My issue is fixed by v0.11. Please close this issue on your own time.

**Comment by imjasonh** [comment]
> @imjasonh Thanks for releasing v0.11. My issue is fixed by v0.11. Please close this issue on your own time.

There's actually a fix still needed, which will become v0.11.1
